.npmignore

```diff
@@ -0,0 +1 @@
+src
```

Comment:
I tried to `npm install` , but npm ignoring `src` prevents you from building `src` after `npm install` so I just `cloned` and built instead 😄 .  

Do we want to exclude `src` from our distribution? I suppose `lib` will be built in our release branch?